java/ql/src/Security/CWE/CWE-200/TempDirLocalInformationDisclosure.ql

```diff
@@ -102,6 +103,21 @@ private class FileCreateTempFileSink extends FileCreationSink {
   }
 }
 
+/**
+ * A guard that checks what OS the program is running on.
```

Comment:
Isn't this only checking if the OS is Windows (based on the subclasses below), and not in general checking for any OS?

Reply:
Yes, good call, I'll update this